server/src/main/java/org/opensearch/action/admin/cluster/health/ClusterHealthRequest.java

```diff
@@ -90,6 +94,10 @@ public ClusterHealthRequest(StreamInput in) throws IOException {
         }
         waitForNoInitializingShards = in.readBoolean();
         indicesOptions = IndicesOptions.readIndicesOptions(in);
+        if (in.getVersion().onOrAfter(Version.V_3_0_0)) {
+            awarenessAttribute = in.readOptionalString();
+            level = in.readEnum(Level.class);
```

Comment:
3.0 will require modifying this to 2.5 when you backport and then again 3.0 to main. You can start with 2.5 and then directly backport, will be faster